### PR TITLE
Added bulk file upload download urls for spender

### PR DESCRIPTION
**Pull request by generalpy101**
Added new urls to generate s3 files upload and download urls for spender.

### CODE REVIEW

File: src/spender/paths/spender@files@generate_urls@bulk.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            properties:
+              data:
+                type: array
+                items:
+                  $ref: ../../components/schemas/file.yaml#/file_generate_urls_action_out
+                example:
+                  - id: sdfd2391
+                    name: uber_expenses_vmrpw.pdf
+                    content_type: application/pdf
+                    download_url: https://aaa.bbb.cc/x232sds
```
Comment (kartikeyrajvaidya) [inline src/spender/paths/spender@files@generate_urls@bulk.yaml:43]: we decided to have both upload and download URLs no?

File: src/spender/paths/spender@files@generate_urls@bulk.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      content:
+        application/json:
+          schema:
+            type: object
+            additionalProperties: False
+            properties:
+              data:
+                type: array
+                items:
+                  $ref: ../../components/schemas/file.yaml#/file_generate_urls_action_out
```
Comment (kartikeyrajvaidya) [inline src/spender/paths/spender@files@generate_urls@bulk.yaml:38]: how will this work?
`file_generate_urls_action_out` doesn't have `thumbnail_200_200_download_url`

**Review by Code0987 — APPROVED** [review]
lgtm

File: src/admin/paths/admin@files@generate_urls@bulk.yaml [hunk 3]
```diff
@@ -42,11 +42,15 @@ post:
                     content_type: application/pdf
                     download_url: https://aaa.bbb.cc/x232sds
                     upload_url: https://aaa.bbb.cc/x232sds
+                    thumbnail_200_200_download_url: https://aaa.bbb.cc/x232sds
+                    thumbnail_200_200_upload_url: https://aaa.bbb.cc/x232sds
```
Comment (kartikeyrajvaidya) [inline src/admin/paths/admin@files@generate_urls@bulk.yaml:46]: this is not required

File: src/admin/paths/admin@files@generate_urls@bulk.yaml [hunk 4]
```diff
@@ -42,11 +42,15 @@ post:
...
                   - id: sdfd2392
                     name: lyft_expenses_vmrpw.pdf
                     content_type: application/pdf
                     download_url: https://aaa.bbb.cc/x22ddsds
                     upload_url: https://aaa.bbb.cc/x2sdssds
+                    thumbnail_200_200_download_url: https://aaa.bbb.cc/x2sdssds
+                    thumbnail_200_200_upload_url: https://aaa.bbb.cc/x2sdssds
```
Comment (kartikeyrajvaidya) [inline src/admin/paths/admin@files@generate_urls@bulk.yaml:53]: this is not required

File: src/components/schemas/file.yaml [hunk 5]
```diff
@@ -61,6 +61,16 @@ file_generate_urls_action_out:
       type: string
       description: |
         Link to download this file
+    thumbnail_200_200_download_url:
+      nullable: true
+      type: string
+      description: |
+        Link to download thumbnail related to this file
+    thumbnail_200_200_upload_url:
```
Comment (kartikeyrajvaidya) [inline src/components/schemas/file.yaml:69]: this is not required

**Pull request merged**